docs/architecture/cloud-native/combine-containers-serverless-approaches.md

```diff
@@ -28,9 +28,12 @@ func init ProjectName --docker
 
 Choose which worker runtime you want from the following options:
 
-- `dotnet` (C#)
-- `node` (JavaScript)
-- `python`
+- `Dotet` (C# or F#)
```

Comment:
typo use `dotnet`

Reply:
Since, we're focusing on dotnet through out this book, I suggest to use the dotnet runtime. I couldn't modify the unedited text here, so I created a gist for your reference: https://gist.github.com/nishanil/27687b0a6255d46c87c66c528c0e0f23

Reply:
Fixed. Guess each platform should be capitalized? 'Dotnet,' not 'dotnet'??